tox.ini

```diff
@@ -21,11 +25,9 @@ commands = pytest -v --tb native -s {posargs} {toxinidir}/tests/unit
 
 [testenv:integration]
 deps =
-    # Until 2.8.6 is released
```

Comment:
Should we switch this to the pypi version instead of git? I don't have the context on this, but the comment seems to be saying that we're using the git version until pypi can be updated.

Reply:
Yes, we can drop this now.  Sidecar charms may still need to use the branch for [this fix](https://github.com/juju/python-libjuju/pull/504) but other than that, PyPI should be sufficiently up to date.